src/ctelnet.cpp

```diff
@@ -1139,16 +1139,6 @@ QString cTelnet::getNewEnvironLanguage()
     return mudlet::self()->getInterfaceLanguage();
 }
 
-QString cTelnet::getNewEnvironFont()
```

Comment:
Isn't this something out of scope for *this* PR?

Reply:
We determined elsewhere this is accessible from client side Mudlet already. It was never used.